controllers/authorino_controller_test.go

```diff
@@ -417,7 +448,10 @@ func newFullAuthorinoInstance() *api.Authorino {
 			EvaluatorCacheSize:       &cacheSize,
 			Listener: api.Listener{
 				Tls: api.Tls{
-					Enabled: &tlsDisabled,
+					Enabled: &tslEnable,
+					CertSecret: &k8score.LocalObjectReference{
+						Name: testCertSecretName,
+					},
```

Comment:
With the default tested authorino instance now having tls enabled at listener and oidc server, the deployment for this configuration is now tested by the exisitng test at:
https://github.com/Kuadrant/authorino-operator/blob/db4674db97df4a9fe6eb8a835098357b63524580/controllers/authorino_controller_test.go#L123-L162

And the container args for the oidc server tls cert and key path is tested at:
https://github.com/Kuadrant/authorino-operator/blob/db4674db97df4a9fe6eb8a835098357b63524580/controllers/authorino_controller_test.go#L484-L486